pkg/build/builder/sti.go

```diff
@@ -265,7 +265,7 @@ func (s *S2IBuilder) Build() error {
 		} else {
 			glog.V(2).Infof("No push secret provided")
 		}
-		glog.V(1).Infof("Pushing %s image ...", pushTag)
+		glog.V(0).Infof("Pushing image %s ...", pushTag)
```

Comment:
would like to see this loglevel issue sorted out before merging because ideally i'd like to reserve "0" for warnings/errors and "1" for info we always want to show.